config/app.php

```diff
@@ -30,4 +30,5 @@
 
     'charset' => env('CHARSET', ini_get('php.output_encoding') ?: ini_get('default_charset') ?: 'UTF-8'),
 
+    'name' => 'librenms',
```

Comment:
We have a config value of `project_id`. Should we use that and if we do, as it's configurable, will that be an issue?

Reply:
I tried setting it in `misc/config_definitions.json` and it does not like it there.

```
        "name": {
            "default": "librenms",
            "type": "string"
        },
```

Setting that and running `lnms config:clear` just results as it showing up as `laravel`.

May be a way around this. Sitting down to dig into how Laravel talks to Monolog.

Reply:
```
    protected function createSyslogDriver(array $config)
    {
        return new Monolog($this->parseChannel($config), [
            $this->prepareHandler(new SyslogHandler(
                Str::snake($this->app['config']['app.name'], '-'),
                $config['facility'] ?? LOG_USER, $this->level($config)
            ), $config),
        ], $config['replace_placeholders'] ?? false ? [new PsrLogMessageProcessor()] : []);
    }
```
 in `vendor/laravel/framework/src/Illuminate/Log/LogManager.php`.
 
Changing it to `app.name` and clearing the config cache did not change anything. Appears to need to be set in that file.

Reply:
Ahh! Set it via env!

Reply:
Done! :)

Reply:
Now controllable via the `.env` file.

```
    'name' => env('APP_NAME', 'librenms'),
```

Reply:
The problem with LibrenmsConfig is it is not loaded when this file is process and this file is also cached.
I don't think we need to speculate about configuration of name, if the need arises then someone can send a PR.